include/ignition/gui/resources.qrc

```diff
@@ -23,4 +23,8 @@
   <file>qml/images/export_icon.png</file>
   <file>qml/images/search.svg</file>
 </qresource>
+<qresource prefix="ign-gui-qml/ignition/gui">
```

Comment:
Can we add a comment here that we should add files here if they're meant to be used by downstream?

Reply:
Added a comment in 26698357a3976e165f8a515afd3ce60fde1f6fbe but can add more explanation if needed. Let me know if you have any suggestions on additional information to include in the comment.